### PR TITLE
use authTokenService for session quotas restrictions

### LINKED ISSUE #15360: Session quotas needs to be replaced with user auth token quotas
Currently session quotas are implemented by counting the number of records in session storage, [code](https://github.com/grafana/grafana/blob/master/pkg/services/quota/quota.go#L32-L39).

We need to replace that with looking at stored user auth tokens instead.

### PR DESCRIPTION
**Pull request by bergquist**
closes #15360

### CODE REVIEW

File: pkg/services/auth/auth_token_test.go [hunk 1]
```diff
@@ -31,6 +31,12 @@ func TestUserAuthToken(t *testing.T) {
 			So(userToken, ShouldNotBeNil)
 			So(userToken.AuthTokenSeen, ShouldBeFalse)
 
+			Convey("Can count active tokens", func() {
```
Comment (bergquist) [inline pkg/services/auth/auth_token_test.go]: should have tests that excludes old tokens as well.

File: pkg/services/auth/auth_token.go [hunk 2]
```diff
@@ -35,6 +35,19 @@ func (s *UserAuthTokenService) Init() error {
 	return nil
 }
 
+func (s *UserAuthTokenService) ActiveAuthTokenCount() (int64, error) {
+
+	tokenMaxLifetime := time.Duration(s.Cfg.LoginMaxLifetimeDays) * 24 * time.Hour
```
Comment (marefr) [inline pkg/services/auth/auth_token.go]: Duplicate code which are used in `LookupToken` and almost also in token_cleanup.go. Should be able to extract some common parts.

File: pkg/services/auth/auth_token.go [hunk 3]
```diff
@@ -35,6 +35,19 @@ func (s *UserAuthTokenService) Init() error {
 	return nil
 }
 
+func (s *UserAuthTokenService) ActiveAuthTokenCount() (int64, error) {
```
Comment (marefr) [inline pkg/services/auth/auth_token.go]: A bit picky maybe but my OCD is too strong. All other methods in this service are referred to by `Token` and not `AuthToken`.

**Review by marefr — CHANGES_REQUESTED** [review]
Looks good :+1: Some minor comments

**Pull request merged**